### PR TITLE
README + remove unnecessary script

### PR DESCRIPTION
**Pull request by jam13**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,41 @@
...
+Keys must be listed in a `multikey` file located in the pipeline folder of the s3 secrets bucket (e.g. `s3://secrets-bucket/my-pipeline/multikey`). 
+
+An example multikey file:
+
+```
+bitbucket healthforge/my-repo
+github acme/another-repo
+```
+
+Private keys should then be placed in a `multikey` folder also within the pipeline folder. Key filenames
```
Comment (jam13) [inline README.md:19]: Not sure it's a good idea to have a file and folder with the same name. I know it's valid in S3, but it feels wrong.

**Comment by YarekTyshchenko** [comment]
I guess I should have copied the readme section from here: https://github.com/healthforge/buildkite-key-rotator#multikey

File: README.md (hunk 1 above)
Comment (YarekTyshchenko) [reply README.md:19]: Hmm, I suppose it is weird. Got any good name suggestions in mind? `multikeyfile` ?

**Comment by YarekTyshchenko** [comment]
Rebased and combined the readme contents. Looks good. i'm going to merge it

**Review by YarekTyshchenko — APPROVED** [review]
:+1:

**Pull request merged**